.gitignore

```diff
@@ -24,6 +24,6 @@ wiki
 !template.latex
 
 # But do ignore these:
-
 /Shared/gitrev.h
 /priiloader/data/certs.bin
+priiloader/data/loader.bin
```

Comment:
If you want that to be an absolute path in the repository you need to prefix it with / (like the two paths above). Otherwise it becomes some kind of wildcard.